lib/p2p/Peer.ts

```diff
@@ -255,6 +416,102 @@ class Peer extends EventEmitter {
 
     this.emit('error', { msg, err });
   }
+
+  private sendHello = (): HelloPacket => {
+    // TODO: use real values
+    const packet = new HelloPacket({
+      version: '123',
+      nodeKey: '123',
+      listenPort: 20000,
+      pairs: ['BTC/LTC'],
+    });
+
+    this.sendPacket(packet);
+
+    return packet;
+  }
+
+  private handleHello = (packet: HelloPacket): void => {
+    this.handshakeState = { ...this.handshakeState, ...packet.body };
+
+    // TODO: define XUD_MIN_VERSION, throw exception if version below
+  }
+
+  private sendPing = (): PingPacket => {
+    this.lastPingId = uuidv1();
+    const packet = new PingPacket({ id: this.lastPingId! });
+
+    this.sendPacket(packet);
+
+    return packet;
+  }
+
+  private handlePing = (packet: PingPacket): void  => {
+    const { id } = packet.body;
+
+    if (!id) {
+      return;
+    }
+
+    this.sendPong(id + 1);
```

Comment:
Why `id + 1` and not just `id`? `id` shouldn't be altered as it is compared to the original value (`this.lastPingId`) or am I missing something?

Reply:
Oops, I did it to make the pong fail, when I was testing it, and forgot to remove it.
Thanks for catching this ;)